app/views/admin/users/_form.html.erb

```diff
@@ -1,26 +1,34 @@
-<% if @user.new_record? && Pageflow.config.quotas.get(:users, @user.account).exhausted? %>
-  <%= render 'admin/users/quota_exhausted' %>
-<% else %>
-  <%= admin_form_for([:admin, @user.becomes(User)]) do |f| %>
-    <%= f.inputs "Details" do %>
-      <% if @user.new_record? %>
-        <%= quota_state_description(:users, @user.account) %>
-      <% end %>
-
-      <%= f.input :email, :hint => f.object.new_record? && I18n.t('pageflow.admin.users.email_invitation_hint') %>
-      <%= f.input :first_name %>
-      <%= f.input :last_name %>
-      <%= f.input :locale,
-                  as: :select,
-                  include_blank: false,
-                  collection: available_locales_collection %>
+<%= admin_form_for([:admin, @user.becomes(User)]) do |f| %>
+  <%= f.inputs "Details" do %>
+    <% if @user.new_record? %>
+      <%= quota_state_description(:users, Pageflow::AccountPolicy::Scope
+                                  .new(current_user, Pageflow::Account).member_addable.first) %>
```

Comment:
For now, I just render quota state descriptions for all member addable accounts, because quota handling will later be treated again anyway